unstructured/metrics/evaluate.py

```diff
@@ -77,11 +88,13 @@ def measure_text_extraction_accuracy(
             fn_txt = fn + ".txt"
 
         if fn_txt in source_list:  # type: ignore
-            output_cct = elements_to_text(elements_from_json(os.path.join(output_dir, doc)))
-            source_cct = _read_text(os.path.join(source_dir, fn_txt))
+            try:
+                output_cct = _prepare_output_cct(os.path.join(output_dir, doc), output_type)
+                source_cct = _read_text(os.path.join(source_dir, fn_txt))
+            except Exception:
+                pass
```

Comment:
this might be too generous; we at least should bring the exception message up so the log have a record on if any doc has issues with processing